packages/account/src/test-utils/launchNode.ts

```diff
@@ -3,13 +3,11 @@ import { randomBytes } from '@fuel-ts/crypto';
 import { FuelError } from '@fuel-ts/errors';
 import type { SnapshotConfigs } from '@fuel-ts/utils';
 import { defaultConsensusKey, hexlify, defaultSnapshotConfigs } from '@fuel-ts/utils';
-import type { ChildProcessWithoutNullStreams } from 'child_process';
 import { randomUUID } from 'crypto';
 import { existsSync, mkdirSync, rmSync, writeFileSync } from 'fs';
 import os from 'os';
 import path from 'path';
 import { getPortPromise } from 'portfinder';
-import treeKill from 'tree-kill';
```

Comment:
I note the `tree-kill` dep is still in the [fuels `package.json`](https://github.com/FuelLabs/fuels-ts/blob/638eae3c3bea11ad2f016c1757e2e014e89d5c2a/packages/fuels/package.json#L87) but is it being used there? If not we should remove it there as well as in the [knip.json](https://github.com/FuelLabs/fuels-ts/blob/638eae3c3bea11ad2f016c1757e2e014e89d5c2a/.knip.json#L31)

Reply:
Done in the last commits.